src/network/raknet/queue/priority/PriorityQueue.ts

```diff
@@ -0,0 +1,46 @@
+import { Priorities } from "./Priorities";
+
+// PROTOTYPE ONLY
+class PriorityItem {
```

Comment:
Maybe this could just be an interface?

```ts
enum Priority {
 // priorities
}
interface PriorityItem {
	item: any;
	priority: Priority;
}
```

Reply:
So, should i keep the first prototype or the second one?